test/integ/cli/job_templates.py

```diff
@@ -0,0 +1,137 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+Job template utilities for incremental download tests.
+"""
+
+from pathlib import Path
+from typing import Optional
+
+from deadline.client.api._submit_job_bundle import create_job_from_job_bundle
+from deadline.client.config import config_file
+
+
+def get_job_bundle_path(template_name: str) -> str:
+    """
+    Get the path to a job bundle template directory.
+
+    Args:
+        template_name: Name of the template (e.g., 'make_many_small_files')
+
+    Returns:
+        Path to the template directory
+    """
```

Comment:
tiniest nit: these are job bundles, not just the templates, so template_name feels a bit weird

```suggestion
def get_job_bundle_path(name: str) -> str:
    """
    Get the path to a job bundle directory.

    Args:
        name: Name of the job bundle (e.g., 'make_many_small_files')

    Returns:
        Path to the job bundle directory
    """
```

Reply:
Sure will do :))

Reply:
Done